lib/common/memory/src/madvise.rs

```diff
@@ -1,7 +1,6 @@
 //! Platform-independent abstractions over [`memmap2::Mmap::advise`]/[`memmap2::MmapMut::advise`]
 //! and [`memmap2::Advice`].
 
-use std::fs::File;
```

Comment:
maybe unecessary on Mac but breaks the build for everyone else :)

Reply:
It was indeed 'unused' on macOS because it's only usage is gated in

```
    #[cfg(any(
        target_os = "linux",
        target_os = "freebsd",
        target_os = "android",
        target_os = "fuchsia",
        target_os = "emscripten",
        target_os = "wasi",
        target_env = "uclibc",
    ))]
```

I've moved the import into that scope to resolve the problem.

https://github.com/qdrant/qdrant/pull/6357/commits/72cbf76f39a5eea3cc120b5c6d6094e9c827b0a8